site/content/en/docs/manual/advanced/consensus.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Consensus'
+linkTitle: 'Consensus'
+weight: 26
+description: 'Consensus based annotation'
+---
+
+By involving multiple annotators, consensus-based annotation incorporates a range of perspectives and interpretations, which helps in minimising individual biases that might skew the annotations.
+
+Multiple annotators reviewing and agreeing on annotations can help identify and correct errors or inconsistencies that might arise from individual biases.
+
+For example creating bounding boxes around the face of a Basset Hound dog. One person might consider the ears also as the part of dogs face and one might not, this is due to fact that different people would have different understanding of things around them. In such cases, consensus based approach helps in reducing this bias by considering the interpretation through consensus.
+
+| | |
+|---|---|
+|![basket_hound_ear_covered](/images/basket_hound_ear_covered.png)|![basket_hound_ear_not_covered](/images/basket_hound_ear_not_covered.png)|
+
+
+See:
+
+- [Use Cases](#use-cases)
+- [Terminology:](#terminology)
+- [Step by Step Guide](#step-by-step-guide)
+
+## Use Cases
+
+1. Instead of annotating the entire dataset with multiple annotators, you can use a smaller set of highly skilled annotators and rely on their consensus to create high-quality ground truth annotations. This reduces the overall cost compared to annotating every instance with multiple annotators.
```

Comment:
It doesn't sound like a valid use case.

```suggestion
The basic use case is annotating the entire dataset with multiple annotators to obtain high-quality annotations in the dataset. In some cases it may work, but typically, however, this way of annotating is prohibitively expensive, as you need to annotate everything several times. There are several ways how the situation can be improved.

1. You can use several highly skilled annotators and rely on their consensus to create high-quality Ground Truth annotations. Typically, the Ground Truth subset contains a small portion of the whole dataset (e.g. 2-5%), so it's relatively easy to annotate it several times. These annotations are used then to validate work of different annotators randomly, so it's very desirable to avoid single expert bias in Ground Truth, and this is one of the key benefits of the consensus annotation. This method requires extra work, but it's much lower than annotating all the dataset several times. For instance, annotating a 3% of images 5 times will result in extra 3% * (5 - 1) = 12% overhead, comparing to single GT annotations.
```